src/tests/blend_tests.cpp

```diff
@@ -22,47 +22,58 @@ static constexpr uint32_t kDefaultDMAColorChannel = 9;
+    {"0", NV097_SET_BLEND_FUNC_SFACTOR_V_ZERO},
+    {"1", NV097_SET_BLEND_FUNC_SFACTOR_V_ONE},
+    {"srcRGB", NV097_SET_BLEND_FUNC_SFACTOR_V_SRC_COLOR},
+    {"1-srcRGB", NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_SRC_COLOR},
+    {"srcA", NV097_SET_BLEND_FUNC_SFACTOR_V_SRC_ALPHA},
+    {"1-srcA", NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_SRC_ALPHA},
+    {"dstA", NV097_SET_BLEND_FUNC_SFACTOR_V_DST_ALPHA},
+    {"1-dstA", NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_DST_ALPHA},
+    {"dstRGB", NV097_SET_BLEND_FUNC_SFACTOR_V_DST_COLOR},
+    {"1-dstRGB", NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_DST_COLOR},
+    {"srcAsat", NV097_SET_BLEND_FUNC_SFACTOR_V_SRC_ALPHA_SATURATE},
+    {"ctRGB", NV097_SET_BLEND_FUNC_SFACTOR_V_CONSTANT_COLOR},
+    {"1-ctRGB", NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_CONSTANT_COLOR},
+    {"ctA", NV097_SET_BLEND_FUNC_SFACTOR_V_CONSTANT_ALPHA},
+    {"1-ctA", NV097_SET_BLEND_FUNC_SFACTOR_V_ONE_MINUS_CONSTANT_ALPHA},
+};
 
 static constexpr uint32_t kTextureSize = 256;
 static constexpr uint32_t kTexturePitch = kTextureSize * 4;
 
 BlendTests::BlendTests(TestHost &host, std::string output_dir, const Config &config)
     : TestSuite(host, std::move(output_dir), "Blend tests", config) {
-  tests_[kAddName] = [this]() {
-    Test(kAddName, [this]() { Body(NV097_SET_BLEND_EQUATION_V_FUNC_ADD, 
-        NV097_SET_BLEND_FUNC_SFACTOR_V_ONE, NV097_SET_BLEND_FUNC_DFACTOR_V_ONE); });
-  };
-  tests_[kSubtractName] = [this]() {
-    Test(kSubtractName, [this]() { Body(NV097_SET_BLEND_EQUATION_V_FUNC_SUBTRACT,
-        NV097_SET_BLEND_FUNC_SFACTOR_V_ONE, NV097_SET_BLEND_FUNC_DFACTOR_V_ONE); });
-};
-  tests_[kRevSubtractName] = [this]() {
-    Test(kRevSubtractName, [this]() { Body(NV097_SET_BLEND_EQUATION_V_FUNC_REVERSE_SUBTRACT,
-        NV097_SET_BLEND_FUNC_SFACTOR_V_ONE, NV097_SET_BLEND_FUNC_DFACTOR_V_ONE); });
-  };
-  tests_[kMinName] = [this]() {
-    Test(kMinName, [this]() { Body(NV097_SET_BLEND_EQUATION_V_MIN,
-        NV097_SET_BLEND_FUNC_SFACTOR_V_ONE, NV097_SET_BLEND_FUNC_DFACTOR_V_ONE); });
-  };
-  tests_[kMaxName] = [this]() {
-    Test(kMaxName, [this]() { Body(NV097_SET_BLEND_EQUATION_V_MAX,
-        NV097_SET_BLEND_FUNC_SFACTOR_V_ONE, NV097_SET_BLEND_FUNC_DFACTOR_V_ONE); });
-  };    
-  tests_[kSignedAddName] = [this]() {
-    Test(kSignedAddName, [this]() { Body(NV097_SET_BLEND_EQUATION_V_FUNC_ADD_SIGNED,
-        NV097_SET_BLEND_FUNC_SFACTOR_V_ONE, NV097_SET_BLEND_FUNC_DFACTOR_V_ONE); });
-  };
-  tests_[kSignedRevSubtractName] = [this]() {
-    Test(kSignedRevSubtractName, [this]() { Body(NV097_SET_BLEND_EQUATION_V_FUNC_REVERSE_SUBTRACT_SIGNED,
-        NV097_SET_BLEND_FUNC_SFACTOR_V_ONE, NV097_SET_BLEND_FUNC_DFACTOR_V_ONE); });
-  };
+  for (const auto &test_eqn : kBlendEqns) {
+    std::string name = "";
+    uint32_t factor = kBlendFactors[1].value;
+    name += kBlendFactors[1].name;
+    name += "_";
+    name += kBlendFactors[1].name;
+    name += "_";
+    name += test_eqn.name;
```

Comment:
I think it'd be more readable to have the equation first or in between the src and dst factors.